pkg/controller/gitopsconfig/controller.go

```diff
@@ -75,8 +76,22 @@ func add(mgr manager.Manager, r reconcile.Reconciler) error {
 		return err
 	}
 
+	// sigs.k8s.io/controller-runtime v0.1.10 is used now
+	// If the version were at least 0.2.0, a ready predicate.GenerationChangedPredicate could be used instead of writing this one
+	// https://godoc.org/sigs.k8s.io/controller-runtime/pkg/predicate#GenerationChangedPredicate
```

Comment:
[optional] Would you be OK with rewording the comment above to a more direct "call to action" style, with explicit condition at start to make it slightly easier to see up front when the requested action should be done? e.g.:

    // TODO: once we update to sigs.k8s.io/controller-runtime >=0.2.0, use their 
    // .../pkg/predicate.GenerationChangedPredicate instead of rewriting it on our own